rustls/src/crypto/aws_lc_rs/mod.rs

```diff
@@ -174,6 +174,12 @@ static SUPPORTED_SIG_ALGS: WebPkiSupportedAlgorithms = WebPkiSupportedAlgorithms
         webpki_algs::RSA_PKCS1_2048_8192_SHA256_ABSENT_PARAMS,
         webpki_algs::RSA_PKCS1_2048_8192_SHA384_ABSENT_PARAMS,
         webpki_algs::RSA_PKCS1_2048_8192_SHA512_ABSENT_PARAMS,
+        #[cfg(all(feature = "aws-lc-rs-unstable", not(feature = "fips")))]
```

Comment:
It occurs to me that the `fips` feature isn't great for additivity, because we're treating it as a feature but in this context it actually shrinks the API surface. Bit of a risk, but might not be much we can do?